core/controllers/contributor_dashboard_admin.py

```diff
@@ -516,7 +516,8 @@ class ContributorDashboardAdminStatsHandlerNormalizedPayloadDict(TypedDict):
     language_code: Optional[str]
     sort_by: Optional[str]
     topic_ids: Optional[List[str]]
-    max_days_since_last_activity: Optional[int]
+    start_date_locale_string: str
```

Comment:
No need for the locale_string suffix
`locale` - what does this mean in this context
`string` - can be deducted from the schema, so no need to duplicate types

Look at other dates in the project, what is the most common pattern they use?

Reply:
done